msiafterburner/tools/chocolateyInstall.ps1

```diff
@@ -4,6 +4,8 @@ $checksum = '42B257623C9445D5BC5EEDDD44DA8CC885C43A16FD2A98077338F937B777EAA3'
 $checksumtype = 'sha256'
 $unpackDir = $(Split-Path -parent $MyInvocation.MyCommand.Definition)
 $unpackFile = Join-Path $unpackDir 'afterburner.zip'
+$pp = Get-PackageParameters
+$RTSSdesktopShortcutPath = "$env:USERPROFILE\Desktop\RTSS.lnk"
```

Comment:
Would it make more sense for this to use `$env:PUBLIC` instead of `$env:USERPROFILE` so that the shortcut appears on all users' desktops on a multi-account system?

Reply:
Thanks, i've updated path to PUBLIC.